lib/cisco_node_utils/bgp_af.rb

```diff
@@ -398,9 +422,15 @@ def dampening=(damp_array)
         CiscoLogger.debug("Dampening 'dampening #{damp_array.join(' ')}''")
       elsif route_map.is_a? String
         # 'dampening route-map WORD' command
-        route_map = "route-map #{damp_array}"
-        route_map.strip!
-        CiscoLogger.debug("Dampening 'dampening #{route_map}'")
+        if platform == :nexus
+          route_map = "route-map #{damp_array}"
+          route_map.strip!
+          CiscoLogger.debug("Dampening 'dampening #{route_map}'")
+        elsif platform == :ios_xr
+          route_policy = "route-policy #{damp_array}"
+          route_policy.strip!
+          CiscoLogger.debug("Dampening 'dampening #{route_policy}'")
```

Comment:
I was confused by the check against route_map above (see comment to Rich). You can disregard this. :-)